### PR TITLE
Release 3.36.0 : Fixes Client Telemetry Release Test

### PR DESCRIPTION
**Pull request by sourabh1007**
## Description

Client Telemetry Release Test was added as part of PR https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3900, during that time, client telemetry service URL was configured as environment variable and for Integration and emulator tests, we had corresponding URL configured.

But during PR https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4050 when environment variable was removed at that time emulator test were fixed but it broke integration test as assertion code of these tests is common and No url was configured for Integration test.

We don't run integration tests as part of PR pipeline, hence it never got caught. Now found this issue while running release pipeline where we run this integration test also.

## Type of change
- [] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by philipthomas-MSFT — DISMISSED** [review]

**Review by kirankumarkolli — DISMISSED** [review]

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/ClientTelemetryTestsBase.cs [hunk 1]
```diff
@@ -15,19 +15,17 @@ namespace Microsoft.Azure.Cosmos.SDK.EmulatorTests
...
-    using Documents.Rntbd;
     using System.Globalization;
     using System.Linq;
     using Cosmos.Util;
     using Microsoft.Azure.Cosmos.Telemetry.Models;
 
     public abstract class ClientTelemetryTestsBase : BaseCosmosClientHelper
     {
-        protected static readonly Uri telemetryServiceEndpoint = new Uri("http://dummy.telemetry.service/api/url");
+        protected static readonly Uri telemetryServiceEndpoint = new Uri("https://tools.cosmos.azure.com/api/clienttelemetry/trace");
```
Comment (ealsur) [inline Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/ClientTelemetryTestsBase.cs:28]: Should this be a config setting or are we ok with this being public?

Comment (sourabh1007) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/ClientTelemetryTestsBase.cs:28]: yes..anyway anybody can see it using fiddler or similar tool.

Comment (ealsur) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/ClientTelemetryTestsBase.cs:28]: If the url is only used for Release pipeline, why not use a pipeline variable and avoid it in the code?

**Review by ealsur — DISMISSED** [review]

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/ClientTelemetryTestsBase.cs (hunk 1 above)
Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/ClientTelemetryTestsBase.cs]: IMO its not needed. These are public endpoints anyways.

**Review by philipthomas-MSFT — APPROVED** [review]

**Pull request merged**